docs/zkapps/tutorials/01-hello-world.mdx

```diff
@@ -189,17 +208,19 @@ First, open `src/Square.ts` in your editor, then add the following at the top of
 7 } from 'snarkyjs';
 ```
 
-What each of these are:
+These fields are:
```

Comment:
are these Field types? 

I'm curious about how we write: Field vs field

- Field types is accurate
- Fields is inaccurate (since a literal is not Fields plural)
- fields seems right, but not literal